### PR TITLE
added event batch create api

### PR DESCRIPTION
**Pull request by shankaragoudab**

### CODE REVIEW

File: course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/EventsActor.java [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+public class EventsActor extends BaseActor {
+    private EventBatchDao eventBatchDao = new EventBatchDaoImpl();
+    private String dateFormat = "yyyy-MM-dd";
+    private String timeZone = ProjectUtil.getConfigValue(JsonKey.SUNBIRD_TIMEZONE);
+    private List<String> validCourseStatus = Arrays.asList("Live", "Unlisted");
+    private UserOrgService userOrgService = UserOrgServiceImpl.getInstance();
+
+    @Inject
+    @Named("course-batch-notification-actor")
+    private ActorRef courseBatchNotificationActorRef;
```
Comment (karthik-tarento) [inline course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/EventsActor.java:45]: No need of this one

File: service/app/controllers/coursemanagement/CourseBatchController.java [hunk 2]
```diff
@@ -132,4 +136,20 @@ public CompletionStage<Result> updateStartBatchesStatus(Http.Request httpRequest
                 null,
                 httpRequest);
     }
+
+    public CompletionStage<Result> createEventBatch(Http.Request httpRequest) {
```
Comment (karthik-tarento) [inline service/app/controllers/coursemanagement/CourseBatchController.java]: this should be in it's own new controller for event

File: course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java [hunk 3]
```diff
@@ -837,4 +839,65 @@ private boolean batchDatesUpdateNotificationActive() {
             PropertiesCache.getInstance()
                     .getProperty(JsonKey.SUNBIRD_BATCH_UPDATE_NOTIFICATIONS_ENABLED));
   }
+
+  private void createEventBatch(Request actorMessage) throws Throwable {
```
Comment (karthik-tarento) [inline course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java]: is this code required ??

**Review by karthik-tarento — CHANGES_REQUESTED** [review]
Plenty of variables still has "courseBatch" in it's name -- just rename those to eventBatch

File: course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/EventsActor.java (hunk 1 above)
Comment (shankaragoudab) [reply course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/EventsActor.java:45]: it is used int the last line 305

File: course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java (hunk 3 above)
Comment (shankaragoudab) [reply course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java]: not required, i have removed (earlier code)

**Pull request merged**